docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
       - POSTGRES_PORT=${POSTGRES_PORT}
     ports:
-      - "5432:5432"
+      - "127.0.0.1:5432:5432"
```

Comment:
ports:
      - "5432:5432"
    port open for all in cloud

 - "127.0.0.1:5432:5432"
internal port seem host 

no need  exposing POSTGRES containers for all

Reply:
Oh, I see, in that case, we can use the hostname for accessing Postgres rather than exposing ports by keeping all in one network. Can you look into it

Reply:
its work  like magic :)
i try this